docs/orchestrating-elastic-stack-applications/security/auth-configs-using-stack-config-policy.asciidoc

```diff
@@ -0,0 +1,20 @@
+:page_id: auth-config-using-stack-config-policy
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Authentication policies using Elastic Stack configuration policy
```

Comment:
I'm wondering if the idea of "authentication policy" makes sense here. 

```suggestion
= Managing authentication realms and role mapping using Elastic Stack configuration policy
```

Reply:
Yes I think we should avoid the concept of authentication policies. It will only confuse users. I think when we introduced it internally we thought we would have a separate CRD for it but that is not the path we went down.

Reply:
```suggestion
= Managing authentication for multiple stacks using Elastic Stack configuration policy
```
just an idea